blocks/blockoracle_test.go

```diff
@@ -369,4 +367,5 @@ func TestMinerBlockOracle_GetEligibleLayers(t *testing.T) {
 		}
 	}
 	r.Equal(eligibleLayers, len(blockOracle.GetEligibleLayers()))
+
```

Comment:
?

Reply:
I have an enabled `gofumpt` file watcher, so it does auto-formatting when I change files. When I changed the signature of `GetNodeAtxIDForEpoch` the watcher auto-formatted changed files. Now I reverted that commit, so some formatting fixes were reverted as well